onadata/apps/api/viewsets/floip_viewset.py

```diff
@@ -43,6 +45,39 @@ def build_json_resource_obj(cls, fields, resource, resource_instance,
         return obj
 
 
+class FLOIPResponsePageNumberPagination(JsonApiPageNumberPagination):
+
+    def get_paginated_response(self, data, descriptor_id: str):
+        nxt = None
+        previous = None
+
+        if self.page.has_next():
+            nxt = self.page.next_page_number()
+        if self.page.has_previous():
+            previous = self.page.previous_page_number()
+
+        data_relationships: dict = data["relationships"] \
+            if data.get("relationships") else {}
```

Comment:
can we replace the ternary operator with `data.get("relationships", {})`

Reply:
Changed in the latest commit